testlapack/matgen.go

```diff
@@ -283,6 +283,31 @@ func Dlagge(m, n, kl, ku int, d []float64, a []float64, lda int, rnd *rand.Rand,
 	// superdiagonals to ku.
 }
 
+// dlarnv fills dst with random numbers from a uniform or normal distribution
+// specified by dist:
+//  dist=1: uniform[0,1),
```

Comment:
Yea, it's not exactly a useful or necessary piece of information. I will change it to parentheses